docs/reference/buildx_imagetools_inspect.md

```diff
@@ -123,13 +123,11 @@ Manifests:
 
 #### JSON output
 
-A `json` go template func is also available if you want to render fields as
-JSON bytes:
+A `json` template function is also available if you want to render fields in
+JSON format:
 
 ```console
 $ docker buildx imagetools inspect crazymax/loop --format "{{json .Manifest}}"
-```
-```json
```

Comment:
I don't have a really strong preference but I like to highlight JSON output

![image](https://github.com/docker/buildx/assets/1951866/7f2810a9-5045-4efc-bada-f5ef83435390)